spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 $LOAD_PATH.unshift('./lib/')
 
 ProcessStatusStub = Struct.new(:exitstatus)
-$STATUS_ZERO = ProcessStatusStub.new(0)
-$STATUS_ONE = ProcessStatusStub.new(1)
+STATUS_OK = ProcessStatusStub.new(0)
+STATUS_ERROR = ProcessStatusStub.new(1)
```

Comment:
I think STATUS_ZERO etc is correct, as this is the return status for an open3 function, and I don't think we'd tie context to the error codes at that level